src/main/java/org/ohdsi/circe/cohortdefinition/LocationRegion.java

```diff
@@ -0,0 +1,19 @@
+package org.ohdsi.circe.cohortdefinition;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import org.ohdsi.analysis.versioning.CdmVersion;
+
+@JsonTypeName("LocationRegion")
+@CdmVersion(semver = ">=6.1")
+public class LocationRegion extends GeoCriteria {
```

Comment:
LocationRegionCriteria

Reply:
I see that 99% of criteria aren't postfixed with `Criteria`: `ConditionEra`, `DeviceExposure`, `Observation`...